cloud_experts_tutorials/cloud-experts-consistent-egress-ip.adoc

```diff
@@ -70,33 +85,40 @@ $ oc get node -o json | \
   "capacity": 14
 }
 ---
-
++
 [NOTE]
 ====
-The above example uses `jq` as a friendly filter. If you do not have `jq` installed, you can review the `metadata.annotations['cloud.network.openshift.io/egress-ipconfig']` field of each node manually to verify node capacity.
+The example uses `jq` as a filter. If you do not have `jq` installed, review the `metadata.annotations['cloud.network.openshift.io/egress-ipconfig']` field of each node manually to verify node capacity.
 ====
 
-== Create the egress IP rule(s)
+== Creating the egress IP rules
 
-=== Identify the egress IPs
+=== Identifying the egress IPs
```

Comment:
I think you should restructure this section as a procedure. Each of the headers would change to a step with a number